RCS-CSM_DIGITAL_AUTOPILOT.s

```diff
@@ -295,7 +295,7 @@ TENTHSEK	TS	ATTSEC
 #	    ALSO IGNORE AUTOMATIC STEERING
 #	    SET = + BY
 #		1) INITIALIZATION PHASE OF AUTOPILOT
-#		2) OCCURANCE OF RHC COMMANDS
+#		2) OCCURRENCE OF RHC COMMANDS
```

Comment:
This is `OCCURANCE` in the scan of [page 1007](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1007.jpg).